Dungeoneer/assets/data/animations.dat

```diff
@@ -1,4 +1,5 @@
 {
+    "$schema": "http://delverengine.com/jsonschema/current/filetypes/animations.dat",
```

Comment:
I'm getting the following IDE error: `Property $schema is not allowed.`

Reply:
Correct. Because the current deployed versions of the schemas do not allow the property. 😄 
This will be resolved after merge and updating the live schema.

Reply:
Ah, my bad. Okay then!